src_modules/voice_commands/index.ts

```diff
@@ -93,17 +92,49 @@ export default class VoiceCommandsModule extends Module {
         this.registerConfigEntry(this.tokenConfigEntry);
 
         this.voiceCommandsEnabledConfigEntry = new BooleanGuildConfigEntry({
-            name: "voiceCommandsEnabled",
-        }, bot.database, true);
-        this.registerConfigEntry(this.voiceCommandsEnabledConfigEntry);
+                            name: "voiceCommandsEnabled",
+                        }, bot.database, true);
+                        this.registerConfigEntry(this.voiceCommandsEnabledConfigEntry);
+                
+                this.autoJoinConfigEntry = new BooleanGuildConfigEntry({
+                            name: "autoJoin",
+                        }, bot.database, false);
+                        this.registerConfigEntry(this.autoJoinConfigEntry);
+        
+                this.autoJoinEnabledConfigEntry = new BooleanGuildConfigEntry({
```

Comment:
This is missing the property definition, add it. Also the duplicate auto join config entry is not needed, remove the autoJoin one and keep the autoJoinEnabled.